README.md

```diff
@@ -48,22 +47,32 @@ var TryStardust = React.createClass({
 module.exports = TryStardust;
 ```
 
-## Develop
+## Installation
+Start by cloning this repo and installing dependencies:
+
+    git clone git@github.com:TechnologyAdvice/stardust.git stardust && cd $_
```

Comment:
That's three lines instead of two. I like to teach people while they're reading and this does that. Is this really worth an update in your eyes @levithomason?